VAMobile/src/components/WaygateWrapper.tsx

```diff
@@ -45,7 +46,7 @@ export const WaygateWrapper: FC<WaygateWrapperProps> = ({ children, waygateName,
   }
 
   const waygate = waygateEnabled(waygateScreen as WaygateToggleType)
-  if (waygate.enabled === false && waygateTypeCheck(waygate.type) && (waygate.errorMsgTitle || waygate.errorMsgBody)) {
+  if (isFocused && waygate.enabled === false && waygateTypeCheck(waygate.type) && (waygate.errorMsgTitle || waygate.errorMsgBody)) {
```

Comment:
This conditional is getting pretty gnarly, but it's going to be refactored in this [PR](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7453), so I left it here for now